packages/react/src/primitives/Accordion/types.ts

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { ElementType, PrimitiveProps, BaseViewProps } from '../types/view';
+
+export interface BaseAccordionProps extends BaseViewProps {
+  /**
+   * @description
+   * The value of the item(s) to expand.  Use on uncontrolled component.
+   */
+  defaultValue?: string[];
```

Comment:
While this would increase complexity in the implementation, a DX improvement could be to conditionally type `defaultValue`/`value` based on whether `allowMultiple` is `true`. IMO the array is extraneous in the default use case